app/controllers/payment-links/get-start.controller.js

```diff
@@ -3,9 +3,19 @@
 const lodash = require('lodash')
 
 const { response } = require('../../utils/response.js')
+const { getCurrentCredential } = require('../../utils/credentials')
 
 module.exports = (req, res) => {
   lodash.set(req, 'session.pageData.createPaymentLink', {})
 
-  return response(req, res, 'payment-links/index', {})
+  const credential = getCurrentCredential(req.account)
+
+  const accountUsesWorldpayMotoMerchantCode = (credential && credential.payment_provider === 'worldpay'
+      && credential.credentials && credential.credentials.merchant_id
```

Comment:
(minor) Could use `lodash.get(credential, 'credentials.merchant_id', '')` to reduce few conditions but fine as it is.

Reply:
Good idea, will do.